### DISCUSSION TITLE
Sticky note cursor jumps to end after typing 1 char

### DISCUSSION THREAD

**Discussion by zacjones93** [johnlindquist/kit#1165] [Errors]
```
kit: "1.51.7",
os: macOS Monterey version 12.1 (21C52)
Apple M1 Max
```

Is this the intended behavior for the sticky note? 👇🏻 

![Screen Recording 2023-03-15 at 02 43 24 PM](https://user-images.githubusercontent.com/6188161/225411971-e5e17926-f7fe-47f8-a995-4f9c449aa1a8.gif)

After using the feature most days for the past 6 months I don't recall this being the behavior here.

**Comment by Ambushfall** [comment]
Hello!

This can be temporarily remedied by commenting out the ~/.kit/main/sticky.js line 10

This is most likely the result of an onInput handler re-setting it's state for the editor, not 100% sure though.

[Code](https://github.com/johnlindquist/kit/search?q=scrollTo)

```js
// Name: Sticky Pad
// Description: Take Quick Notes
let stickyPath = kenvPath("sticky.md");
let contents = await ensureReadFile(stickyPath, `
# Sticky Notes
`.trim());
let changed = false;
contents = await editor({
    value: contents,
    // scrollTo: "bottom", [LINE 10] << Reason for the sticky notes issue
    // footer: `Escape to save to ${stickyPath}`,
    shortcuts: [
        {
            name: "Save and Close",
            key: "escape",
            onPress: async (input, { inputChanged }) => {
                changed = inputChanged;
                await submit(input);
                await hide();
            },
            bar: "right",
        },
    ],
    onAbandon: async (input, { inputChanged }) => {
        changed = inputChanged;
        submit(input);
    },
    onInput: async () => {
        changed = true;
    },
});
if (changed)
    await writeFile(stickyPath, contents + "\n");
export {};
//# sourceMappingURL=sticky.js.map

```

**Reply by johnlindquist** [reply]
@Ambushfall Fun side-note: you can also add a "comma-handler.js" script in your main scripts dir if you want to change the behavior of pressing "," from the main menu.

You can see all the handlers you can override in the ~/.kit/handlers dir
